css/style.css

```diff
@@ -569,12 +688,77 @@ body {
     margin-right: 0%;
   }
 
+  .project-popup {
+    visibility: visible;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    position: fixed;
+    background: rgba(0, 0, 0, 0.6);
+    border-radius: 15px;
+    border: 1px solid #dfe1e6;
+  }
+
+  .project-container {
+    width: 70%;
+    border: 1px solid #dfe1e6;
+    background: #fff;
+    padding: 0% 3% 5% 3%;
+    border-radius: 15px;
+    margin-left: 12%;
+    margin-right: 12%;
+    margin-top: 5%;
+    filter: none;
+  }
```

Comment:
- [ ] As requested by the previous reviewer please make sure that the popup fills the screen (both desktop and mobile) as shown in the Figma design as this is how your popup currently looks:
    - Your design (Desktop):
      ![image](https://user-images.githubusercontent.com/104968236/231468510-86cfc6a0-9db5-48b5-b4ca-07df66e215a1.png)
    - Your design (Mobile):
      ![image](https://user-images.githubusercontent.com/104968236/231468801-bf35befe-f773-4efd-b874-80fccd1447ce.png)
- As you can see above part of the popup is getting clipped so please make sure you fix that 💪.